src/entropy.rs

```diff
@@ -41,8 +50,6 @@ impl fmt::Display for EntropyError {
     }
 }
 
-impl std::error::Error for EntropyError {}
```

Comment:
don't delete the instance indiscriminately, you should just hide it behind the feature flag. (that also goes for the other instances)

Reply:
Okay, done, I've put it behind a feature flag, though I suspect you're making this whole thing more hardcore for no_std. Go ahead and close the PR if you see fit!